src/bandersnatch/master.py

```diff
@@ -22,9 +22,12 @@ class XmlRpcError(aiohttp.ClientError):
 
 
 class Master:
-    def __init__(self, url: str, timeout: float = 10.0) -> None:
+    def __init__(
+        self, url: str, timeout: float = 10.0, global_timeout: float = 5 * 60 * 60.0
```

Comment:
```suggestion
        self, url: str, timeout: float = 10.0, global_timeout: float = 5.0 * 60 * 60
```
Total NIT and don't care.